graphistry/feature_utils.py

```diff
@@ -2606,3 +2599,40 @@ def _featurize_or_get_edges_dataframe_if_X_is_None(
             reuse_if_existing=True,
             memoize=memoize,
         )
+
+    def _features_by_col(self, column_part: str, kind: str):
+        if kind == 'nodes' and hasattr(self, '_node_features'):
+            X = self._node_features
+        elif kind == 'edges' and hasattr(self, '_edge_features'):
+            X = self._edge_features
+        else:
+            raise ValueError('make sure to call `featurize` or `umap` before calling `get_features_by_cols`')
+        
+        transformed_columns = X.columns[X.columns.map(lambda x: True if column_part in x else False)]  # type: ignore
+        return X[transformed_columns]  # type: ignore
+    
+    def get_features_by_cols(self, columns: Union[List, str], kind: str = 'nodes'):
```

Comment:
Maybe helpful for the docs, if I wanted all features generated for input column `"src_ip"`, could I use this? (Are original column names used for the prefix of generated feature column names?)

Either way, maybe useful to clarify here. 

I keep wondering every time I see this helper.. how to understand feature provenance..